Modules/_json.c

```diff
@@ -305,15 +305,8 @@ static void
 raise_errmsg(const char *msg, PyObject *s, Py_ssize_t end)
 {
     /* Use JSONDecodeError exception to raise a nice looking ValueError subclass */
-    _Py_static_string(PyId_decoder, "json.decoder");
-    PyObject *decoder = _PyImport_GetModuleId(&PyId_decoder);
-    if (decoder == NULL) {
-        return;
-    }
-
-    _Py_IDENTIFIER(JSONDecodeError);
-    PyObject *JSONDecodeError = _PyObject_GetAttrId(decoder, &PyId_JSONDecodeError);
-    Py_DECREF(decoder);
+    PyObject *JSONDecodeError =
+        _PyImport_GetModuleAttrString("json.decoder", "JSONDecodeError");
```

Comment:
This is a potentially hot code path in something like a ``try-except`` block with json decoding inside.

Is there some way to avoid the cost of converting the `C const char *` to python unicode objects?

Reply:
Yeah, See my comment on the issue https://github.com/python/cpython/issues/90699#issuecomment-1298452495. We agreed on using static strings in this module and `asyncio`.